src/Event/OrganisationUserIdEvent.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\os2forms_organisation\Event;
+
+use Drupal\Component\EventDispatcher\Event;
+
+/**
+ * Organisation user id event.
+ */
+class OrganisationUserIdEvent extends Event {
+  const EVENT_NAME = 'organisation_user_id_event';
```

Comment:
Is this constant ever used?

Reply:
Nope, it was added while i was following https://www.drupal.org/docs/develop/creating-modules/subscribe-to-and-dispatch-events. It seems it was never necessary.